test/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.12.9 as builder
+FROM golang:1.13.3 as builder
```

Comment:
kubebuilder version on line 9 is still on v1 (that binary might not even be needed)

Reply:
Good catch. It's needed because kubebuilder includes an API server we're running against.

Reply:
Fixed.